zcash_keys/src/address.rs

```diff
@@ -143,7 +143,11 @@ impl UnifiedAddress {
 
     /// Returns whether this address has a Sapling receiver.
     pub fn has_sapling(&self) -> bool {
-        self.sapling.is_some()
```

Comment:
Document this in a `Fixed` section, as we will be pushing this out in a `0.1.1`.

Reply:
Fixed.